compiler/ast/src/expressions/ternary.rs

```diff
@@ -31,7 +31,7 @@ pub struct TernaryExpression {
 
 impl fmt::Display for TernaryExpression {
     fn fmt(&self, f: &mut fmt::Formatter) -> fmt::Result {
-        write!(f, "if {} ? {} : {}", self.condition, self.if_true, self.if_false)
+        write!(f, "(if {} ? {} : {})", self.condition, self.if_true, self.if_false)
```

Comment:
Ternary expressions in Leo do not contain an `if` keyword.
```suggestion
        write!(f, "({} ? {} : {})", self.condition, self.if_true, self.if_false)
```